gax-httpjson/build.gradle

```diff
@@ -1,7 +1,7 @@
 archivesBaseName = "gax-httpjson"
 
 // TODO: Populate this from dependencies.properties version property (for proper Gradle-Bazel sync)
-project.version = "0.65.1" // {x-version-update:gax-httpjson:current}
+project.version = "0.66.0" // {x-version-update:gax-httpjson:current}
```

Comment:
Why is this one different?

Reply:
I believe the httpjson gax helpers are only for the gapic generator that uses http/json which is not considered GA.